### PR TITLE
move bootstrap to separated script it can be easy versioned

### PR DESCRIPTION
**Pull request by acidsennui**
pt: https://www.pivotaltracker.com/story/show/161817138

### CODE REVIEW

File: scripts/bootstrap.sh [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    DEFAULT=YES
+    shift # past argument with no value
+    ;;
+    *)
+          # unknown option
+    ;;
+esac
+done
+
+cd /infrastructure/ansible
```
[the remark this answers is not in the document]
Comment (acidsennui) [reply scripts/bootstrap.sh]: or cd `pwd`/ansible ?

[the remark this answers is not in the document]
Comment (valkov-ivaylo) [reply scripts/bootstrap.sh]: Nothing. It is simpler actually.

[the remark this answers is not in the document]
Comment (acidsennui) [reply scripts/bootstrap.sh]: cd ../ansible will not work 
```pwd
echo "RUN SCRIPT BOOT"
bash /infrastructure/scripts/bootstrap.sh \
```
```RUN SCRIPT BOOT
/
/infrastructure/scripts/bootstrap.sh: line 34: cd: ../ansible: No such file or directory```

[the remark this answers is not in the document]
Comment (acidsennui) [reply scripts/bootstrap.sh]: interesing is that it is changing when run on reboot


```

/infrastructure
RUN SCRIPT BOOT
/infrastructure
```

[the remark this answers is not in the document]
Comment (dincho) [reply scripts/bootstrap.sh]: I guess this is something to do with your other task about reboot ?

Looking the user_data again, we execute the boostrap with absolute path while PWD is not really change (thus the `/`).

Maybe try `dirname` suggestion by @valkov-ivaylo

File: scripts/bootstrap.sh [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+# Temporary fix/workaround for non-executable vault install
+chmod +x /usr/bin/vault
+
+# Authenticate the instance to CSM
+PKCS7=$(curl -s http://169.254.169.254/latest/dynamic/instance-identity/pkcs7 | tr -d '\n')
+
+export VAULT_ADDR=$vault_addr
+export VAULT_TOKEN=$(vault write -field=token auth/aws/login pkcs7=$PKCS7 role=$vault_role)
+
+export env=$env
```
Comment (dincho) [inline scripts/bootstrap.sh]: why exporting this?

Comment (acidsennui) [reply scripts/bootstrap.sh]: i will check is it working without... but... good question

**Comment by acidsennui** [comment]
closing this one because all change are in https://github.com/aeternity/infrastructure/pull/158/

**Pull request closed without merging**